buildsystem/dependencies.gradle

```diff
@@ -6,10 +6,10 @@ allprojects {
 
 ext {
   //Android
-  androidBuildToolsVersion = "23.0.1"
+  androidBuildToolsVersion = "26.0.2"
   androidMinSdkVersion = 15
-  androidTargetSdkVersion = 21
-  androidCompileSdkVersion = 21
+  androidTargetSdkVersion = 25
+  androidCompileSdkVersion = 25
```

Comment:
I would change them to 26 to be more consistent. 

I think that there were used `21`(much lower to build tools version) just to do not need to implement Android permission runtime for release version of app.